editor/plugins/org.fusesource.ide.projecttemplates/lifecycle-mapping-metadata.xml

```diff
@@ -3,22 +3,35 @@
 <lifecycleMappingMetadata>
 
 	<pluginExecutions>
-		<pluginExecution>
-			<pluginExecutionFilter>
-				<groupId>org.apache.maven.plugins</groupId>
-				<artifactId>maven-compiler-plugin</artifactId>
-				<versionRange>[2.0,)</versionRange>
-				<goals>
-					<goal>compile</goal>
-					<goal>testCompile</goal>
-				</goals>
-			</pluginExecutionFilter>
-			<action>
-				<configurator>
-					<id>org.fusesource.ide.projecttemplates.camel.maven.configurator.wtp</id>
-				</configurator>
-			</action>
-		</pluginExecution>
+        <pluginExecution>
+            <pluginExecutionFilter>
+                <groupId>org.apache.maven.plugins</groupId>
+                <artifactId>maven-compiler-plugin</artifactId>
+                <versionRange>[2.0,)</versionRange>
+                <goals>
+                    <goal>compile</goal>
+                    <goal>testCompile</goal>
+                </goals>
+            </pluginExecutionFilter>
+            <action>
+                <configurator>
+                    <id>org.fusesource.ide.projecttemplates.camel.maven.configurator.wtp</id>
+                </configurator>
+            </action>
+        </pluginExecution>
+        <pluginExecution>
+            <pluginExecutionFilter>
+                <groupId>org.jboss.redhat-fuse</groupId>
+                <artifactId>fabric8-maven-plugin</artifactId>
+                <versionRange>[7.1.0,7.2.0)</versionRange>
+                <goals>
+                    <goal>resource</goal>
```

Comment:
I think it would better to include the same goals than on the "official" lifecycle metadata plugin https://github.com/fabric8io/fabric8-maven-plugin/blob/master/plugin/src/main/resources/META-INF/m2e/lifecycle-mapping-metadata.xml

Reply:
It was only failing on one of those 5 goals - do we need or really want to include them all?

Reply:
it will avoid that projects using the other goals to hit the bug. i think it is better to stay as close as possible than the upstream version.